pkg/spoke/managedcluster/creating_controller.go

```diff
@@ -53,40 +53,80 @@ func NewManagedClusterCreatingController(
 }
 
 func (c *managedClusterCreatingController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
-	_, err := c.hubClusterClient.ClusterV1().ManagedClusters().Get(ctx, c.clusterName, metav1.GetOptions{})
-	switch {
-	case errors.IsUnauthorized(err),
-		errors.IsForbidden(err) && strings.Contains(err.Error(), anonymous):
+	existingCluster, err := c.hubClusterClient.ClusterV1().ManagedClusters().Get(ctx, c.clusterName, metav1.GetOptions{})
+	if err != nil && (errors.IsUnauthorized(err) || errors.IsForbidden(err) && strings.Contains(err.Error(), anonymous)) {
```

Comment:
if errors.IsUnauthorized(err) || errors.IsForbidden(err)

do we really need to check anonymous?

Also pls add a comment here on why we return when unauthorized.

Reply:
ok

Reply:
also use skipUnauthorizedError here

Reply:
> also use skipUnauthorizedError here
@qiujian16 if use skipUnauthorizedError, here will be `if err != nil && skipUnauthorizedError(err) == nil && strings.Contains(err.Error(), anonymous) ` , is it ok?

Reply:
> also use skipUnauthorizedError here

done @qiujian16